# snp_selection
just a repo to track how on earth i am going to even get to the stage of doing MR 

## Selection from DIAMANTE and what we know 
Decided to go for this https://diagram-consortium.org/downloads/DIAMANTE_summary_statistics_download.pdf
- chose the European data as we had discussed to focus on European first
- the data came in a ``.txt`` file - the data contains summary statistics from a GWAS so not individual level data to use PLINK on

To find the right SNPs for MR, they should follow:
- Genome-wide significance threshold (p-value < 5 × 10⁻⁸).
- independent SNPs (minimal linkage disequilibrium).
- Clearly defined effect sizes and allele frequencies from the source dataset.

## then use the ``run.bash`` script 
- 
